packages/dredd/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dredd",
-  "version": "12.2.0",
+  "version": "12.2.1",
```

Comment:
Should this be a patch version? iirc we've removed JSON Schema Draft 3 support

Reply:
`12.2.1` contains a deprecation warning upon using JSON Schema Draft 3, but the support is still there. Since we've discussed to update to the next major Dredd version that would include `gavel@9`, I'll bump these dependencies versions once they are released.